lib/gooddata/bricks/hello_world_brick.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'brick'
+
+module GoodData
+  module Bricks
+    # Simple brick used for testing and debu purposes
+    class HelloWorldBrick < GoodData::Bricks::Brick
+      def version
+        '0.0.1'
+      end
+
+      # HelloWorld brick entry-point
+      #
+      # @param [Hash] params Parameters
+      # @option [String] 'message' text to be returned in result, if nill - nothing is returned
+      def call(params)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)